File: SimpleSchedulerAngular/src/app/models/worker.ts
export type Worker = {
  workerID: number,
  isActive: boolean,
  workerName: string,
  detailedDescription: string,
  emailOnSuccess: string,
  parentWorkerID?: number | null,
  timeoutMinutes: number,
  directoryName: string,
  executable: string,
  argumentValues: string
};

File: SimpleSchedulerDomainModels/Worker.cs
namespace SimpleSchedulerDomainModels;

public record class Worker(
    long ID,
    bool IsActive,
    string WorkerName,
    string DetailedDescription,
    string EmailOnSuccess,
    long? ParentWorkerID,
    int TimeoutMinutes,
    string DirectoryName,
    string Executable,
    string ArgumentValues
);
